test/fixtures/es-class-assign-property/expected-remove-context-wrap-es6.js

```diff
@@ -0,0 +1,23 @@
+class Foo1 extends React.Component {
+  render() {}
+}
+
+process.env.NODE_ENV !== "production" ? Foo1.contextTypes = {
+  bar1: PropTypes.string
+} : void 0;
+
+process.env.NODE_ENV !== "production" ? Foo1.childContextTypes = {
+  bar1: PropTypes.string
+} : void 0;
+
+class Foo2 extends React.PureComponent {
+  render() {}
+}
+
+process.env.NODE_ENV !== "production" ? Foo1.contextTypes = {
```

Comment:
Sorry but I wasn't referring to the typo, I was referring to removing everything.